src/SearchInput/SearchInput.tsx

```diff
@@ -45,6 +50,8 @@ export interface SearchInputProps extends CommonFieldProps {
   resultsBorder?: boolean
   /** optional boolean to enable fuzzy search via fuse.js */
   enableFuzzySearch?: boolean
+  /** optional config of fuzzySearchOptions */
```

Comment:
Should this be:
```suggestion
  /** optional config of fuzzy search */
```

I don't think this can be a config of itself